scheduler/replica_scheduler.go

```diff
@@ -654,6 +658,10 @@ func (rcs *ReplicaScheduler) RequireNewReplica(replicas map[string]*longhorn.Rep
 		return 0
 	}
 
+	if types.IsDataEngineV2(volume.Spec.DataEngine) {
+		return 0
+	}
+
```

Comment:
I missed this check.
What's the purpose of it?

Reply:
This means, we should wait 0s for replenishing a new replica for v2 volumes.